docs/debian-conf.d/main/00_vexim_listmacrosdefs

```diff
@@ -42,3 +42,9 @@ VEXIM_LOCALPART_SUFFIX = +*
 
 CHECK_RCPT_LOCAL_ACL_FILE = /etc/exim4/vexim-acl-check-rcpt.conf
 CHECK_DATA_LOCAL_ACL_FILE = /etc/exim4/vexim-acl-check-content.conf
+CHECK_HELO_LOCAL_ACL_FILE = /etc/exim4/vexim-acl-check-helo.conf
```

Comment:
Maybe this is because they are not parsed directly by the debian magic but included by parsed config files. And it might be confusing if they appear in the same directories. - just a guess.

but on the `acl_check_helo:` i think we don't need a separate .conf file.